src/components/emailerFilterControls/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import moment from 'moment';
+import { EmailerFilters } from '../../containers/email/types';
+import {
+  Collapse,
+  Slider,
+  DatePicker,
+  AutoComplete,
+  Button,
+  message,
+} from 'antd';
+import { SliderMarks } from 'antd/lib/slider';
+import styled from 'styled-components';
+import { Neighborhoods } from '../../assets/content';
+import apiClient from '../../api/apiClient';
+import { CloseOutlined } from '@ant-design/icons';
+
+const StyledCollapse = styled(Collapse)`
+  max-width: 400px;
+`;
+
+const StyledRangePicker = styled(DatePicker.RangePicker)`
+  margin-right: 15px;
+`;
+
+const autoCompleteStyles = { width: 200, marginRight: 15 };
+
+interface EmailerFilterControlsProps {
+  filters: EmailerFilters;
+  setFilters: React.Dispatch<React.SetStateAction<EmailerFilters>>;
+}
+
+const MAX_COUNT = 10;
+
+function activityCountRange(filters: EmailerFilters): [number, number] {
+  return [filters.activityCountMin, filters.activityCountMax || MAX_COUNT + 1];
+}
+
+function formatDates(
+  start?: string,
+  end?: string,
+): [moment.Moment | null, moment.Moment | null] | undefined {
+  if (start && end) return [moment(start), moment(end)];
+}
+
+function disabledDate(current: moment.Moment): boolean {
+  // Can not select future days
+  return current > moment().endOf('day');
+}
+
+const activityCountLabels: SliderMarks = {
+  0: '0',
+  [MAX_COUNT + 1]: `${MAX_COUNT}+`,
```

Comment:
❓Should `${MAX_COUNT}+` be `${MAX_COUNT + 1}+` (for activity counts, `11+` instead of `10+`)?

Reply:
I think it's a matter of preference - I personally read 10+ as "more than 10", which is why right now selecting '10' and '10+' are different, but I can also see it being read as "10 or more". Regardless, I'm happy with either way